hydroflow_lang/src/graph/mod.rs

```diff
@@ -34,6 +38,8 @@ new_key_type! {
 
 /// Context identifier as a string.
 const CONTEXT: &str = "context";
+/// Hydroflow identifier as a string.
+const HYDROFLOW: &str = "df";
```

Comment:
I prefer `hf` for HydroFlow, and to distinguish from the canonical pandas `df` for DataFrame.